diff --git a/src/LocalPath.cpp b/src/LocalPath.cpp
index 7dd8755f63..f6b551d392 100644
--- a/src/LocalPath.cpp
+++ b/src/LocalPath.cpp
@@ -297,7 +297,7 @@ FindDataPaths() noexcept
 #ifndef __APPLE__
   /* Linux (and others): allow global configuration in /etc/xcsoar */
   if (Directory::Exists(Path{"/etc/xcsoar"}))
-    data_paths.emplace_back(Path{"/etc/xcsoar"});
+    result.emplace_back(Path{"/etc/xcsoar"});
 #endif // !APPLE
 #endif // HAVE_POSIX
